Added sid.json

SiddPatki opened the pull request.

SiddPatki committed:
Added sid.json

phenax commented on domains/sid.json:
```diff
@@ -0,0 +1,12 @@
+    "email": "siddharthapatki@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://siddpatki.github.io/my-site/",
```
CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `siddpatki.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

SiddPatki replied:
Sorry and thanks...I am a noob in this so was not sure.
I will update the CNAME as per your comment.
Will I be able to change this CNAME in the future if I want it to redirect to some different link

phenax replied:
You can change the record any time you want with a new pull request

SiddPatki committed:
Updated as per changes by @phenax

CNAME records cannot be a full URL. It has to be a hostname. I'd suggest you rename your repo from https://siddpatki.github.io/my-site/ --->> to siddpatki.github.io

SiddPatki committed:
Merge branch 'main' into main

phenax committed:
Merge branch 'main' into main
